src/nvim/ex_getln.c

```diff
@@ -2374,8 +2641,23 @@ static void draw_cmdline(int start, int len)
     }
 
     msg_outtrans_len(arshape_buf, newlen);
-  } else
-    msg_outtrans_len(ccline.cmdbuff + start, len);
+  } else {
+draw_cmdline_no_arabicshape:
```

Comment:
should we document that coloring is disabled with arabicshape? Or is it a limitiation we expect to remove soon?